Dockerfile

```
@@ -1,8 +1,9 @@
+ARG VERSION=latest
```

will this not bring in node 20?

---

It will bring in non-LTS versions too, if the build argument is not provided (see `docs/guides/Running-In-Production.md`). Many of the changes in this PR are to supply that argument 👍

---

We could use a specific version as the default, instead of `latest` but that could be indirection and cause surprises? 🤔

---

From https://hub.docker.com/_/node there is an `lts` tag, we could use that as the default?
```suggestion
ARG VERSION=lts
```